.travis.yml

```diff
@@ -1,9 +1,6 @@
 sudo: required
-language: java
-jdk:
-  - oraclejdk7
-  - oraclejdk8
-
+language: objective-c
+osx_image: xcode7.3
```

Comment:
Sorry my bad, Xcode 8.3 actually removed support for Swift 2.3. You should be safe on 8.2?